src/components/ContactUs/MobileVersion.tsx

```diff
@@ -52,17 +81,41 @@ export default function MobileVersion() {
           />
           <textarea
             placeholder="Message"
+            name="message"
             className=" inline-block ml-2 h-[270px] resize-none outline-none flex-grow placeholder-[#7E7979] font-robotoMono text-base "
             required
           ></textarea>
         </label>
+        <ValidationError
+          prefix="Message"
+          field="message"
+          errors={state.errors}
+        />
         {/*//TODO: replace this button with the app button component */}
-        <button
-          type="submit"
-          className="border-2 border-[#12A04E] w-full rounded-md p-2 my-2 font-montserrat font-bold text-[#12A04E]"
-        >
-          Send Message
-        </button>
+
+        <ReCAPTCHA
+          sitekey={process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY!}
+          onChange={handleRecaptchaResult}
+        />
+
+        {captchaComplete ? (
+          <button
+            type="submit"
+            disabled={state.submitting}
+            className="border-2 border-[#12A04E] w-full rounded-md p-2 my-2 font-montserrat font-bold text-[#12A04E]"
+          >
+            Send Message
+          </button>
+        ) : (
+          <button
+            type="submit"
+            disabled
+            className="border-2 border-[#12A04E] w-full rounded-md p-2 my-2 font-montserrat font-bold text-[#12A04E]"
+          >
+            Send Message
+          </button>
+        )}
```

Comment:
Same as above.

Reply:
@antosan made changes as per the review with the latest commit.